clang/lib/Frontend/CompileJobCache.cpp

```diff
@@ -296,7 +296,8 @@ std::optional<int> CompileJobCache::initialize(CompilerInstance &Clang) {
   if (!CacheCompileJob)
     return std::nullopt;
 
-  std::tie(CAS, Cache) = Clang.createCASDatabases();
```

Comment:
This is technically `getOrCreateCASDatabase()`? I think maybe rename this function so you can just call it here, then you don't need the new APIs to get `Ptr`

Reply:
You're right. I wanted to check that the CAS has already been created (and is correctly installed in the VFS), but I don't mind just renaming this function.